src/net.h

```diff
@@ -874,6 +884,26 @@ class CNode
         assert(false);
     }
 
+    std::string ConnectionTypeToString() const
+    {
+        switch (m_conn_type) {
+        case ConnectionType::INBOUND:
+            return "inbound";
+        case ConnectionType::MANUAL:
+            return "manual";
+        case ConnectionType::FEELER:
+            return "feeler";
+        case ConnectionType::OUTBOUND:
+            return "outbound-full-relay";
+        case ConnectionType::BLOCK_RELAY:
+            return "block-relay-only";
+        case ConnectionType::ADDR_FETCH:
+            return "addr-fetch";
+        }
```

Comment:
Please add `// no default case, so the compiler can warn about missing cases` comment (https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#c-data-structures)

Reply:
ah thanks. done in this PR. I've also introduced other switches without default in #19316 & #19724. I will also add this comment to the others in #19724.

Reply:
added c99b26010eaf4d446eb5118e38dbcc03fabba11c in #19724